bucket/chipmunk.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "2.9.0",
+    "description": "fast logfile viewer that can deal with huge logfiles (>10 GB). It powers a super fast search and is supposed to be a useful tool for developers who have to analyze logfiles.",
+    "homepage": "https://github.com/esrlabs/chipmunk",
+    "license": "Apache 2.0",
+    "url": "https://github.com/esrlabs/chipmunk/releases/download/2.9.0/chipmunk@2.9.0-win64-portable.zip",
+    "hash": "9b93d050704e322891eda14fe255d5cf6a387c38c79839082e77c1df4faadf76",
+    "bin": "chipmunk.exe",
+    "shortcuts": [
+        [
+            "chipmunk.exe",
+            "chipmunk"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/esrlabs/chipmunk"
+    },
```

Comment:
simplify

Reply:
could you explain or suggest what exactly do you mean? I not  so familiar with scoop packages JSON syntax

Reply:
checkver: github

Reply:
@Ash258 done, thanks a lot for suggestion